docs/tutorials/educators/chemistry.ipynb

```diff
@@ -79,7 +79,7 @@
    },
    "outputs": [],
    "source": [
-    "!pip install openfermion openfermioncirq pyscf openfermionpyscf --quiet"
+    "!pip install openfermion==0.11.0 openfermioncirq pyscf openfermionpyscf==0.4 --quiet"
```

Comment:
Optional: I'd leave a TODO with issue number so folks don't have to dig too much to find out the reason for this and whether or not it's ready for cleanup.

Reply:
Thanks for the review!! I will create an issue and assign it to @rmlarose :) 
The TODO would get into the site https://quantumai.google/cirq/tutorials/educators/chemistry - so I'd rather not put a TODO in a cell there.